src/pysofar/__init__.py

```diff
@@ -42,14 +41,14 @@ def __init__(self, custom_token=None):
     # Helper methods
     def _get(self, endpoint_suffix, params: dict = None):
         url = f"{self.endpoint}/{endpoint_suffix}"
-
         if params is None:
             response = requests.get(url, headers=self.header)
         else:
             response = requests.get(url, headers=self.header, params=params)
 
         status = response.status_code
-        data = json.loads(response.text)
+        # data = json.loads(response.text)
```

Comment:
@tcj - I always remove commented out code, or add a comment as to what it's being left in for.

Reply:
@evanShap — thanks — this did cross my mind.  

I believe `response.json()` is a 1:1 equivalent and readability improvement over `json.loads(response.text)` but meant to seek opinions/input.

[According to SO](https://stackoverflow.com/a/58049213) it is that and more (it adds some useful functionality) — so I'll go ahead and remove the commented code.